engine/src/main/java/org/terasology/math/AABB.java

```diff
@@ -46,10 +46,24 @@ private AABB(Vector3f min, Vector3f max) {
         this.max = max;
     }
 
+    /**
+     * Creates a new AABB from the given minimum and maximum points.
+     *
```

Comment:
Can you add an example like if min or max are inclusive or exclusive?

Reply:
Min and max should be inclusive. However, I do not understand what you mean by adding an example for this. What kind of example?